randovania/data/json_data/prime3.txt

```diff
@@ -3859,18 +4725,16 @@ Command Station
 Asset id: 13589229454537797972
 > Dock to Transit Station 2-A; Heals? False
   * Normal Door to Transit Station 2-A/Dock to Command Station
-  > Pickup (Missile Expansion)
-      Grapple Voltage
   > Main Room
-      HyperModeTank and Hyper Grapple
+      After Command Station Phazon Blob
 
 > Dock to Flux Control; Heals? False
   * Normal Door to Flux Control/Dock to Command Station
   > Morph Ball Tunnels
       Trivial
```

Comment:
- Change dock to a Morph Ball Door
- Rename Dock to "Morph Ball Door to Flux Control"
- Connect to Missile Expansion as Trivial
- Connect to Command Vault as Trivial